src/diffusers/models/unets/unet_2d_condition.py

```diff
@@ -599,14 +469,214 @@ def __init__(
             self.conv_act = get_activation(act_fn)
 
         else:
-            self.conv_norm_out = None
-            self.conv_act = None
+            self.conv_norm_out = None
+            self.conv_act = None
+
+        conv_out_padding = (conv_out_kernel - 1) // 2
+        self.conv_out = nn.Conv2d(
+            block_out_channels[0], out_channels, kernel_size=conv_out_kernel, padding=conv_out_padding
+        )
+
+        self._set_pos_net_if_use_gligen(attention_type=attention_type, cross_attention_dim=cross_attention_dim)
+
+    def _check_config(
+        self,
+        down_block_types: Tuple[str],
+        up_block_types: Tuple[str],
+        only_cross_attention: Union[bool, Tuple[bool]],
+        block_out_channels: Tuple[int],
+        layers_per_block: [int, Tuple[int]],
```

Comment:
@yiyixuxu Seems like an invalid type annotation... My linter breaks when viewing this file :(

Reply:
ohh yes! do you want to open a PR?